src/redux/actions/workspaces-actions.ts

```diff
@@ -7,3 +7,10 @@ export const fetchWorkspaces = () => ({
     throw err;
   }),
 });
+
+export const fetchWorkspace = (ws: string) => ({
+  type: ActionTypes.FETCH_WORKSPACE,
+  payload: WorkspacesHelper.getWorkspace(ws).catch((err) => {
+    throw err;
+  }),
```

Comment:
No need for catch and throw
```suggestion
  payload: WorkspacesHelper.getWorkspace(ws),
```

Reply:
Fixed